scrape_books/spiders/books.py

```diff
@@ -0,0 +1,49 @@
+import re
+from urllib.parse import urljoin
+
+import scrapy
+from scrapy.http import Response
+
+
+class BooksSpider(scrapy.Spider):
+    name = "books"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/catalogue/page-1.html"]
+    base = "https://books.toscrape.com/catalogue/"
+    urls = []
+
+    @staticmethod
+    def get_rating(text: str) -> int:
+        number_str = text.split()[-1]
+        number_dict = {"One": 1, "Two": 2, "Three": 3, "Four": 4, "Five": 5}
+
+        return number_dict.get(number_str, None)
```

Comment:
This is redundant because by default is None
```suggestion
        return number_dict.get(number_str)
```